python/cudf/cudf/core/column/decimal.py

```diff
@@ -226,6 +228,17 @@ def as_numerical_column(
     ) -> "cudf.core.column.NumericalColumn":
         return unary.cast(self, dtype)  # type: ignore[return-value]
 
+    def unary_operator(self, unaryop: str) -> ColumnBase:
```

Comment:
How did this work before? Could we use `abs` on a decimal column?

Ideally we should not have to hardcode the supported operators for each type in Python -- I'd rather catch an exception from C++ and re-raise it with a Python-friendly message.